src/main/java/se/sundsvall/relations/api/model/Relation.java

```diff
@@ -0,0 +1,70 @@
+package se.sundsvall.relations.api.model;
+
+import static io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY;
+import static org.springframework.format.annotation.DateTimeFormat.ISO.DATE_TIME;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Null;
+import java.time.OffsetDateTime;
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.format.annotation.DateTimeFormat;
+
+@Data
+@Builder(setterPrefix = "with")
+@Schema(description = "Relation between objects")
+public class Relation {
+	// TODO verify RequiredMode (defaults to AUTO)
```

Comment:
Do what the TODO says?

Either set all `@NotBlank/NotNull` fields to `requiredMode = REQUIRED` or verify if these are set to required automatically, based on the constraint annotations.